libcxx/test/libcxx/depr/depr.default.allocator/allocator.members/address.cxx2a.verify.cpp

```diff
@@ -12,12 +12,8 @@
 // pointer address(reference x) const;
```

Comment:
Might as well rename the file to `address.cxx20.verify.cpp`.

Reply:
Done. Renamed other cxx2a files in the directory as well.

PS I'd normally land things like this as a separate `NFC` commit to keep the actual changes small, but I'm just doing it all in a single commit here. Let me know if you'd also prefer that to be a separate follow-up change.